docs/developer/create_application.md

```diff
@@ -0,0 +1,45 @@
+# Create an application
+
+## In DiracX-Web
+
+The applicatins created here will be available for DiracX-Web and for all the extensions.
+
+### Declare the application
+
+In the file `packages/diracx-web-components/src/components/ApplicationList.ts` you can extend the `applicationList` with your new app. You must provide a name (explicit), the component representing the new app and an icon that will appear in the `Add application` menu. You can also give two functions, `setState` and `getState`, to configure the export and import of your app.
+
+### Code the application
+
+The code of your app should be in `packages/diracx-web-components/src/components/<new-app>`. The new app can and should use what already exist in `@dirac-grid/diracx-web-components`. 
+
+In order to be compatible with the share and import buttons, the application must write its state to the session storage at `<appId>_State`. This slot is read from and written to by the corresponding functions.
+
+💡You can look at `JobMonitor` as an example.
+
+## In an extension
+
+### Declare the application
+
+The application can be declared in the file `src/<extension name>/applicationList.ts`. The declaration keeps what is in `diracx-web` and adds application(s) specific to the extension.
+
+### Code the application
+
+The code should be in the folder `src/<extension name>/components/<app name>`. 
+
+To continue, refer to the [Code the application](#code-the-application) section in **DiracX-Web** above.
```

Comment:
IIUC, this part is already covered in `docs/developer/manage_extension` (which points to `/packages/extension/README`
```suggestion

```